packages/wrangler/src/deploy/deploy.ts

```diff
@@ -1146,27 +1146,49 @@ async function ensureQueuesExist(config: Config) {
 function updateQueueConsumers(config: Config): Promise<string[]>[] {
 	const consumers = config.queues.consumers || [];
 	return consumers.map((consumer) => {
-		const body: PutConsumerBody = {
-			dead_letter_queue: consumer.dead_letter_queue,
-			settings: {
-				batch_size: consumer.max_batch_size,
-				max_retries: consumer.max_retries,
-				max_wait_time_ms: consumer.max_batch_timeout
-					? 1000 * consumer.max_batch_timeout
-					: undefined,
-				max_concurrency: consumer.max_concurrency,
-			},
-		};
+		if (consumer.type === "http_pull") {
+			// Note: It's weird to define a pull consumer in a worker's config
```

Comment:
What does it mean to be "weird" here?

Reply:
By that I mean it seems strange to define this type of consumer in a worker config because http_pull consumers represent a decoupling of queue consumers from workers. But, our Queues toml api is already weird, so I guess we're just leaning into that.